### PR TITLE
Add new ECS variants

### LINKED ISSUE #2164: add a new aws-ecs variant for kernel 6.1
**What I'd like:**
I'd like a new variant for ECS that includes the new 6.1 kernel.

The ECS Agent works to preserve compatibility across its releases, so there's no upstream version to synchronize with in this case; it's still on 1.x versions.

With that in mind, possible options for a successor to `aws-ecs-1` include:
* `aws-ecs-1.1` - add a ".1" to reflect a new revision
* `aws-ecs-2` - increment "1" to reflect a significant difference in the new variant
* `aws-ecs-2022` - switch to year-based numbering to avoid version confusion with the agent

**Any alternatives you've considered:**
Introducing the 5.15 kernel in a future update to the `aws-ecs-1` variant, but this breaks with the compatibility promise we make for in-place updates.

### PR DESCRIPTION
**Pull request by arnaldo2792**
**Issue number:**
Closes #2164

**Description of changes:**

```
packages: add kmod-5.15-nvidia sources

This adds the sources to compile the 515 NVIDIA driver for the 5.15
kernel.  This version only supports the GPU architectures Maxwell,
Pascal, Volta, Turing, Ampere, and forward.  The driver will use the GPU
System Processor (GSP) feature if the underlying hardware supports it
by loading the binary file `/lib/firmware/nvidia/<version>/gsp.bin`.
```

```
variants: add aws-ecs-1.1 variant
```

```
variants: add aws-ecs-1.1-nvidia variant
```

**Testing done:**
- Launched hosts with new ECS variants, and confirm tasks were scheduled in those hosts
- Tested different types of instances except for `p2` instances since those are no longer supported for in this driver:
```
[   32.672564] NVRM: The NVIDIA Tesla K80 GPU installed in this system is
[   32.672564] NVRM:  supported through the NVIDIA 470.xx Legacy drivers
```

**Terms of contribution:**

By submitting this pull request, I agree that this contribution is dual-licensed under the terms of both the Apache License, version 2.0, and the MIT license.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -867,7 +869,15 @@ There are a few important caveats about the provided kdump support:
 
 ### NVIDIA GPUs Support
 Bottlerocket's `nvidia` variants include the required packages and configurations to leverage NVIDIA GPUs.
-The official AMIs for these variants can be used with EC2 GPU-equipped instance types such as: `p2`, `p3`, `p4`, `g4dn`, `g5` and `g5g`.
+Currently, the following NVIDIA driver versions are supported in Bottlerocket:
+
+* 470.X
+* 515.X
```
Comment (mchaker) [inline README.md:875]: Would it be helpful to add the corresponding Bottlerocket versions alongside the NVIDIA driver numbers? 

e.g.
```markdown
470.X: Bottlerocket v0.X - v0.Y
515.X: Bottlerocket v0.A - v0.B
```

**Review by mchaker — APPROVED** [review]
👍

File: README.md [hunk 2]
```diff
@@ -867,7 +869,15 @@ There are a few important caveats about the provided kdump support:
...
+
+The official AMIs for these variants can be used with EC2 GPU-equipped instance types such as: `p2`, `p3`, `p4`, `g3`, `g4dn`, `g5` and `g5g`.
+Some of these instance types are built with hardware that is no longer supported by the 515.X NVIDIA driver.
```
Comment (jpculp) [inline README.md:878]: I might rephrase this as:
```suggestion
Note that older instance types, such as 'p2', are not supported by NVIDIA driver 515.X and above.
```

File: variants/Cargo.toml [hunk 3]
```diff
@@ -2,7 +2,9 @@
 members = [
     "aws-dev",
     "aws-ecs-1",
+    "aws-ecs-1.1",
     "aws-ecs-1-nvidia",
+    "aws-ecs-1.1-nvidia",
```
Comment (jpculp) [inline variants/Cargo.toml:7]: nit. While neither this, nor the above, is quite alphabetical, this might be more consistent with the rest:
```suggestion
    "aws-ecs-1",
    "aws-ecs-1-nvidia",
    "aws-ecs-1.1",
    "aws-ecs-1.1-nvidia",
```
Same goes for the README.md.

**Review by jpculp — APPROVED** [review]

**Comment by arnaldo2792** [comment]
I'm closing this since we are still working on our tooling to improve the release process for ECS variants.
